docs/Glossary.md

```diff
@@ -13,6 +13,18 @@ description: >
 
 The following terms are arranged in the order of their appearance in the actual user workflow.
 
+### Projects
+**A project is a set of related [Data Scopes](#data-scope), the Data Scopes may contain different [Data Entities](#data-entities) from arbitrary [Data Sources](#data-sources) for Metric Computation**
+
+Project was introduced in v0.15 to solve some complex metric computation problems. For example, There is a **Change Failure Rate** in [DORA Metric](../UserManuals/DORA) which requires connecting Incident(issue.type = 'INCIDENT') to its related Deployment(`cicd_task.type = 'DEPLOYMENT'`) to determine whether a Deployment failed or not. Unfortunately, most data sources do not offer this relationship, the only reasonable approach so far is to map an Incident to its Closest Deployment. Without Project, the Incident may be mapped to unrelated Deployment in database.
+
+A project consists of the following elements:
+
+- A set of [Data Scopes](Glossary.md#data-scope), for example, several `repos` pointing to Github Repositories, `boards` pointing to a couple of Jira Scrum Boads and `cicd_scopes` to Jenkins Servers etc.
+- A set of Metrics(currently, only DORA).
```

Comment:
When I read the whole sentence, I had no idea about 'A project consists the following elements: a set of metrics'. 
Do you mean that there's "a set of metrics" as a project's attribute?